PhysicsTools/CondLiteIO/src/RecordWriter.cc

```diff
@@ -70,7 +70,7 @@ void RecordWriter::update(const void* iData, const std::type_info& iType, const
     label = "";
   }
   std::map<std::pair<edm::TypeIDBase, std::string>, DataBuffer>::iterator itFound =
-      idToBuffer_.find(std::make_pair(edm::TypeIDBase(iType), std::string(iLabel)));
+      idToBuffer_.find(std::make_pair(edm::TypeIDBase(iType), std::string(label)));
```

Comment:
clang analyzer complain that `iLabel` could be `null`. So this change uses `label` which is set to empty string in case `iLabel` is null